src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -66,9 +65,10 @@ object DepositHandler {
     val payload = Paths.get(settings.tempDir.toString, id, deposit.getFilename.split("/").last).toFile
     for {
       _ <- assertTempDirHasEnoughDiskspaceMarginForFile(payload)
-      _ <- copyPayloadToFile(deposit, payload)
+      _ <- copyPayloadToFile(deposit, payload) //TODO should file permissions also be set after this action?
       _ <- doesHashMatch(payload, deposit.getMd5)
       _ <- handleDepositAsync(deposit)
+      _ <- FilesPermissionService.changePermissionsForDirectoryAndContent(deposit.getFile, settings.depositPermissions, id) // set file permissions after continued deposit is finished
```

Comment:
I guess Util is more common in our team than Service. Service reminds too much of servers with servlets.